stl/inc/__msvc_all_public_headers.hpp

```diff
@@ -102,6 +102,10 @@
 #include <vector>
 #include <version>
 
+#if _HAS_CXX20 && defined(__clang__)

```

Comment:
In this header conditional inclusion is only for things that will `#error` and blow up a compile - you should be fine to include `<source_location>` unconditionally.

Reply:
I thought that but it didn't work as it has no idea what `consteval` was. I have changed that to use the feature test macro, just haven't pushed it.

Thought it was strange that it needed that though...

Reply:
Oh, I see, the conditional in `<source_location>` is broken.